attest/verifier/config/src/lib.rs

```diff
@@ -69,6 +69,21 @@ impl TrustedMrEnclaveMeasurement {
                 .collect(),
         }
     }
+
+    /// Get the MRENCLAVE measurement.
+    pub fn mr_enclave(&self) -> &[u8; 32] {
```

Comment:
I'm fine making the attributes public if desired since this is more or less a plain old data type.
I probably lean a bit harder on accessors as a reaction from C interfaces providing no guarantees on fields.

Reply:
One of the advantages of the attributes not being public is that they cannot be modified after construction, which is often a nice guarantee - so I support having them.